### PR TITLE
Adding table bulk assertions for icon, label and color

### PR DESCRIPTION
**Pull request by titantwentyone**
```
assertTableBulkActionHasIcon(string $name, string $icon)
assertTableBulkActionDoesNotHaveIcon(string $name, string $icon)
assertTableBulkActionHasLabel(string $name, string $label)
assertTableBulkActionDoesNotHaveLabel(string $name, string $label)
assertTableBulkActionHasColor(string $name, string $color)
assertTableBulkActionDoesNotHaveColor(string $name, string $color)
```

### CODE REVIEW

**Comment by titantwentyone** [comment]
Failing tests due to...

```
 • Tests\src\Notifications\NotificationTest > it can close notifications
   Livewire\Exceptions\MethodNotFoundException 

  Unable to call component method. Public method [close] not found on component: [filament.notifications.http.livewire.notifications]

  at vendor/livewire/livewire/src/ComponentConcerns/HandlesActions.php:144
    140▕ 
    141▕         if (! method_exists($this, $method)) {
    142▕             throw_if($method === 'startUpload', new MissingFileUploadsTraitException($this));
    143▕ 
  ➜ 144▕             throw new MethodNotFoundException($method, $this::getName());
    145▕         }
    146▕ 
    147▕         throw_unless($this->methodIsPublicAndNotDefinedOnBaseClass($method), new NonPublicComponentMethodCall($method));
    148▕ 

      +26 vendor frames 
  27  tests/src/Notifications/NotificationTest.php:157
      Livewire\Testing\TestableLivewire::call()
```

**Comment by danharrin** [comment]
Merge changes from 2.x and the test should pass :)

File: packages/tables/src/Testing/TestsBulkActions.php [hunk 1]
```diff
@@ -223,6 +223,126 @@ public function assertTableBulkActionDisabled(): Closure
...
+            $this->assertTableBulkActionExists($name);
+
+            $livewire = $this->instance();
+            $livewireClass = $livewire::class;
+
+            $action = $livewire->getCachedTableBulkAction($name);
+
+            Assert::assertTrue(
+                $action->getLabel() === $label,
+                "Failed asserting that a table action with name [{$name}] has label [{$label}] on the [{$livewireClass}] component for record [{$record}]."
```
Comment (danharrin) [inline packages/tables/src/Testing/TestsBulkActions.php:279]: Please ensure all messages read "table bulk action" not "table action"

**Review by danharrin — CHANGES_REQUESTED** [review]

File: packages/tables/src/Testing/TestsBulkActions.php (hunk 1 above)
Comment (titantwentyone) [reply packages/tables/src/Testing/TestsBulkActions.php]: Done. Cheers!

**Comment by danharrin** [comment]
Thanks

**Pull request merged**